bats/helpers/user.bash

```diff
@@ -95,8 +95,7 @@ user_update_username() {
   )
   exec_graphql "$token_name" 'user-update-username' "$variables"
   num_errors="$(graphql_output '.data.userUpdateUsername.errors | length')"
-  username="$(graphql_output '.data.userUpdateUsername.user.username')"
-  [[ "$num_errors" == "0" || "$username" == "$token_name" ]] || exit 1
```

Comment:
why this change? does not seems to be related to the PR

Reply:
This function was broken and was caching a null value. It is required that we cache the correct username for knowing an ln address we can send to.